### PR TITLE
move to nodejs v6.2.2

### PR DESCRIPTION
**Pull request by abresas**
fixes streams error https://github.com/nodejs/node/pull/7160

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -38,7 +38,7 @@ RUN apt-get update \
 		wget \
 	&& rm -rf /var/lib/apt/lists/*
 
-ENV NODE_VERSION 6.2.0
+ENV NODE_VERSION 6.2.2
 ENV NPM_VERSION 3.8.9
```
Comment (Page-) [inline Dockerfile]: Can you update npm to 3.9.5 please, matching what is bundled with nodejs 6.2.2

Comment (abresas) [inline Dockerfile]: sure, just saw your message about npm and was about to do the same thing

Comment (abresas) [inline Dockerfile]: actually, @Page- maybe you meant 3.10.1? https://github.com/npm/npm/blob/master/CHANGELOG.md#v3101-2016-06-17

Comment (Page-) [inline Dockerfile]: https://nodejs.org/en/blog/release/v6.2.2/
> npm: upgrade npm to 3.9.5 (Kat Marchán) #7139

**Comment by Page-** [comment]
LGTM

**Pull request merged**